### DISCUSSION TITLE
distinguish script symbol for local vs instantiated?

### DISCUSSION THREAD

**Discussion by BaddRadish** [godotengine/godot-proposals#5941] [Editor]
so in the scene tree, when we've instantiated a scene, we see the script symbol, but theres no way to tell if the script for that scene comes from the instantiated scene or is something we've attached locally. might be a good idea to gray out or blue out either the scripts that are local or the scripts that come from the instantiated scene so that we can distinguish them at a glance. the scene icons that let you quickly jump to editing the scene should match the instantiated color.

**Comment by theraot** [comment]
Note: Godot 4 will "blue out" scripts that are running in the editor.
